### PR TITLE
Development

### PR DESCRIPTION
**Pull request by jpbatz**
adds time conversion and geo coordinate retrieval to processIncidents.js

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+config.json
```
Comment (keokilee) [inline .gitignore:1]: :+1:

File: processIncidents.js [hunk 2]
```diff
@@ -1,7 +1,78 @@
 // processIncidents.js
 // processes each incident in incidents JSON object
+// ================================================
+var bodyParser = require('body-parser');
```
Comment (keokilee) [inline processIncidents.js]: I don't think you need this, so try running this without this line.

File: processIncidents.js [hunk 3]
```diff
@@ -1,7 +1,78 @@
...
+    }
+  });
+
+  //    (A) store incident to db storeIncident.js
+
+
+// Helper Functions
+// ================
+
+function convertEpocToLocalTime(utcSeconds) {
```
Comment (keokilee) [inline processIncidents.js]: Nitpick: should be convertEpochToLocalTime

**Comment by keokilee** [comment]
Please see comments.

**Comment by jpbatz** [comment]
removed body-parser and corrected spelling of epoch

**Comment by keokilee** [comment]
Did you mean to close this? I can re-open it.

File: processIncidents.js [hunk 4]
```diff
@@ -1,7 +1,78 @@
...
+  var addr = "Antarctica";
+
+  // var geo_coord = getGeoCode(addr);
+  // console.log(geo_coord);
+
+  // console.log("Lat = " + geo_coord.lat);
+  // console.log("Lng = " + geo_coord.lng);
+  
+  // get geo coordinates from GeoCode API
+  request('http://open.mapquestapi.com/geocoding/v1/address?key=Fmjtd%7Cluu8216zl1%2Cax%3Do5-94250w&location='+addr, function (error, response, body) {
```
Comment (keokilee) [inline processIncidents.js]: Uh oh, one more thing. You don't want to commit your key to the repository. If this is your key, generate a new one and add it to your (ignored) config.json.

**Comment by jpbatz** [comment]
MBA removes key in processIndcident.js and replaces it with new one in config.json, clean up code and comments.

**Comment by keokilee** [comment]
Looks good. Time to start geocoding some Hawaii addresses! :earth_americas:

**Pull request merged**